scripts/get-release-artifact-and-commit-sha.js

```diff
@@ -29,6 +29,8 @@ async function main() {
     }
   });
 
+  fs.writeFileSync('latest-commit', tag.commit.sha);
```

Comment:
Why use a file instead of stdout to pass that info around ? 🤔

Reply:
Cause I'm unsure how to feed stdout to a groovy var (I ain't a very good groovy dancer let's say)

So.... Yeah. readFile groovy func looked nice.